src/methods/clipping/difference.jl

```diff
@@ -27,7 +27,7 @@ GI.coordinates.(diff_poly)
 ```
 """
 function difference(
-    geom_a, geom_b, ::Type{T} = Float64; target::Type{Target} = Nothing,
+    geom_a, geom_b, ::Type{T} = Float64; target::Target = nothing,
 ) where {T <: AbstractFloat, Target <: Union{Nothing, GI.AbstractTrait}}
     return _difference(Target, T, GI.trait(geom_a), geom_a, GI.trait(geom_b), geom_b)
 end
```

Comment:
Why doesn't the type on line 39 need to change to `::GI.PolygonTrait`, given it will be an object passed and not a type per say? (Sorry it won't let me comment on those lines for some reason)

Reply:
It's passing `Target` which is the type of `target`.